crawler/lookingglass_crawler/spiders/media_spider.py

```
@@ -0,0 +1,107 @@
+import scrapy
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy_patchright.page import PageMethod
+from ..items import MediaItem
+
+class MediaSpider(CrawlSpider):
+    name = 'media_spider'
+    
+    def __init__(self, *args, **kwargs):
+        super(MediaSpider, self).__init__(*args, **kwargs)
```

Ruff (0.11.9)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)